statements/2025-09-05-activitypub-and-atproto-discourse.md

```diff
@@ -1,3 +1,68 @@
+
+There is an entire section of the ActivityPub specification that isn't as well known or widely adopted but which, at a high level, provides fairly similar ideas to those emphasized within the AT Protocol community for separation between data, identity and applications. Recently, a taskforce within the Social Web Community Group has been established to advance what is now known as the ActivityPub API. 
+
+Whilst we may have our differences at present, over time those gaps will narrow, as we share a lot more in common than we have differences.
+
+There does not have to be a “winning” protocol. We do not build a better open social web for everyone by fighting and arguing about protocol superiority. That is _**not**_ how we achieve a better open social web. Instead, we must work together, cross-pollinate and share ideas, and participate within each other's communities with respect and mutual understanding. Arguing between us only emboldens those that seek to derail and destroy efforts to build an open social web.
+
+The practice of collaboration outside of our own groups has a long history within the standards community, whether that is with competing companies working together on standards or protocols, or collaboration between different standards bodies like the W3C and IETF.
+
+There has already been cross-pollination of ideas between the people working on ActivityPub and AT Protocol. For example, AT Protocol adopted an [internet draft](https://www.ietf.org/archive/id/draft-parecki-oauth-client-id-metadata-document-03.html) that was originally written to support the ActivityPub ecosystem, and projects within ActivityPub have adopted some ideas on [content labeling](https://github.com/swicg/activitypub-trust-and-safety/issues/84) and [starter packs](https://fedidevs.com/starter-packs/) from the AT Protocol ecosystem.
+
+Both ActivityPub and AT Protocol can and do co-exist. This co-existence is perhaps best emphasized by the outstanding work of [Bridgy Fed](https://fed.brid.gy/) project, which connects ActivityPub, AT Protocol, and other protocols together allowing for interoperability and community that crosses between protocols. If you wanted to summarise this letter on a t-shirt, it would be “People > Protocols > Platforms”.
+
+This statement is a call for cooling the temperature of discussions and a reminder to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, which only emboldens and empowers those who do not want either protocol to succeed.
+
+This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://github.com/swicg/meetings/tree/main/2025-09-05), and has been reviewed by several members of the CG.
+
+### Co-signed by the following community members:
```

Comment:
-1

We don't usually sign decisions made by the CG. If this is a position paper agreed to by consensus by the CG, it should be published without signatures.

Reply:
I'd add that even if the CG were to issue a statement and include signatures, it should be limited to CG participants.